src/backend/api/buckets.py

```diff
@@ -320,16 +324,18 @@ async def set_bucket_tagging(
     :return: Returns `True` on success, otherwise `False`.
     """
     async with conn.conn() as s3:
-        tag_set: TaggingTypeDef = {"TagSet": tags}
         try:
-            await s3.put_bucket_tagging(Bucket=bucket_name, Tagging=tag_set)
+            await s3.put_bucket_tagging(
+                Bucket=bucket_name, Tagging=tag_set.dict()  # pyright: ignore
```

Comment:
we should strive to disable specific types warnings, instead of blanket ignoring them all. That way we can catch potential issues introduced later on that would otherwise be ignored.